A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_ProgressPage.js

```diff
@@ -67,8 +96,8 @@ class CDUProgressPage {
             CDUProgressPage.ShowPredictiveGPSPage(mcdu);
         };
         mcdu.setTemplate([
-            ["{green}ECON " + flightPhase + "{end} " + flightNo],
-            ["\xa0" + flightPhase, "REC MAX\xa0", "OPT"],
+            [flightPhase + "{end} " + flightNo],
```

Comment:
```suggestion
            [flightPhase + " " + flightNo],
```

Reply:
Thanks for your review! I will commit these changes.